### DISCUSSION TITLE
Project update

### DISCUSSION THREAD

**Discussion by noobyco** [surani-hub/lazyport#5] [General]
Letz talk about progress...

**Comment by noobyco** [comment]
Added Endpoint creation issue for @surani-hub to work.. btw take ur time haha

**Comment by noobyco** [comment]
> BRANCH 

#### I have created a `dev` branch, @surani-hub make it the default branch. 

- we will keep `dev` branch for development &
- `master` branch for production

**Reply by surani-hub** [reply]
@Blaize15 set the "dev" branch as the default branch.

**Reply by noobyco** [reply]
Awesome @surani-hub

**Comment by noobyco** [comment]
README.md file has been updated with `codespaces` link and a general contribution workflow.
